hackathon/fix-an-issue/README.md

```diff
@@ -0,0 +1,48 @@
+# Fixing an issue in the MDAnalysis core library
+
+With over [400 issues to deal with](https://github.com/MDAnalysis/mdanalysis/issues)
+we are always looking for new and old contributors to help enhance
+and maintain the [MDAnalysis core library](https://github.com/MDAnalysis/mdanalysis).
+
+
+## Team
+
+The MDAnalysis core developers team (@IAlibay, @lilyminium, @fiona-nauthon,
```

Comment:
I just added everyone for now, but maybe that's not a good idea

Reply:
Having names here is good. We'll be helping anyway.